### PR TITLE
[Core] Adding missing methods to python on strategies

### PR DESCRIPTION
**Pull request by loumalouomega**

### CODE REVIEW

File: kratos/solving_strategies/strategies/residualbased_linear_strategy.h [hunk 1]
```diff
@@ -610,12 +610,35 @@ class ResidualBasedLinearStrategy
     /**
      * @brief This method directly sets the input as the LHS
      * @param A The LHS matrix
+     * @todo Remove if unused
      */
     void GetDirectSystemMatrix(TSystemMatrixType& A)
```
Comment (RiccardoRossi) [inline kratos/solving_strategies/strategies/residualbased_linear_strategy.h]: please remove this function. It should not be there! (and i already said this in the review many months back...)

File: kratos/solving_strategies/strategies/residualbased_newton_raphson_strategy.h [hunk 2]
```diff
@@ -820,12 +820,36 @@ class ResidualBasedNewtonRaphsonStrategy
     /**
      * @brief This method directly sets the input as the LHS
      * @param A The LHS matrix
+     * @todo Remove if unused
      */
     void GetDirectSystemMatrix(TSystemMatrixType& A)
```
Comment (RiccardoRossi) [inline kratos/solving_strategies/strategies/residualbased_newton_raphson_strategy.h]: please remove this function

Comment (loumalouomega) [reply kratos/solving_strategies/strategies/residualbased_newton_raphson_strategy.h]: That's why I added the comment

**Comment by loumalouomega** [comment]
Done

**Comment by loumalouomega** [comment]
Done @RiccardoRossi

**Review by RiccardoRossi — APPROVED** [review]

**Pull request merged**